flow-report/Sidebar.tsx

```diff
@@ -0,0 +1,155 @@
+/**
```

Comment:
> `flow-report/Sidebar.tsx`

I know PascalCase is the airbnb style for components, but their style guide would also give us `GatherRunner.js` and `FullPageScreenshot.js` :)

Is the different capitalization scheme desired? I personally would prefer consistency with the rest of the repo, but I'm also not someone doing any work in these files.

Reply:
I'm fine with ditching PascalCase. `Sidebar.tsx` exports more than just `Sidebar` for example. Also test files like `App-test.tsx` look weird.

Reply:
I'm also slight +1 to ditching PascaleCase :)